examples/webpack/webpack.config.js

```diff
@@ -11,6 +11,8 @@ const classNameCompressionMap = require('./class-name-compression-map.json');
 
 const extractCSS = process.env.EXTRACT_TO_CSS === 'true';
 
+console.log('Stylesheet extraction enabled?', extractCSS);
```

Comment:
Is this console.log intentional?

Reply:
oh yeah, i added it as a debugging measure but then decided to keep it as it seemed to be useful

happy to remove it if it's not valuable